bitrise.yml

```diff
@@ -959,23 +949,6 @@ workflows:
         run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
     - git-clone@6.2: {}
     - certificate-and-profile-installer@1.10: {}
-    - script@1.1:
-        title: Pull in L10n
-        inputs:
-        - content: |
-            #!/usr/bin/env bash
-            set -e
-            set -x
-            # Import only the shipping locales (from shipping_locales.txt) on Release
-            # builds. Import all locales on Beta and Fennec_Enterprise, except for pull
-            # requests.
-            git clone https://github.com/mozilla-mobile/ios-l10n-scripts.git ||exit 1
-            pip3 install --user virtualenv
-            cd /usr/local/bin
-            ln -s /Users/vagrant/Library/Python/3.10/bin/virtualenv .
-            cd -
-            virtualenv --version
-            ./ios-l10n-scripts/import-locales-firefox.sh --release
```

Comment:
I can not test this on CI, but in theory, and based on your comment @flodolo we should be able to remove this part too.

This workflow is triggered after the L10nBuild workflow when the process to get the screenshots is run from Taskcluster. We will monitor this next time they run.